modules/yieldmoBidAdapter.js

```diff
@@ -381,7 +387,12 @@ function openRtbRequest(bidRequests, bidderRequest) {
   if (schain) {
     openRtbRequest.schain = schain;
   }
-
+  if (bidRequests[0].transactionId) {
+    openRtbRequest.tid = bidRequests[0].transactionId;
+  }
```

Comment:
You should collect all the transaction ids on imp. Ext. Tid

Reply:
@patmmccann fix it. Thank you.